why-lint-with-commitizen/index.qmd

```diff
@@ -0,0 +1,189 @@
+---
+title: "Why lint with commitizen"
+description: "Reasons to use commitizen to lint our commit messages to follow the Conventional Commits standard."
+date: "2024-10-25"
+categories:
+- develop
+- git
+- standardise
+- workflow
+---
+
+::: content-hidden
+Use other decision posts as inspiration to writing these. Leave the
+content-hidden sections in the text for future reference.
+:::
+
+## Context and problem statement
+
+::: content-hidden
+State the context and some background on the issue, then write a
+statement in the form of a question for the problem.
+:::
+
+Taking time to build automated
+[DevOps](https://en.wikipedia.org/wiki/DevOps) processes can save lots
+of time later that would normally be manually done. For instance,
```

Comment:
Some confusion between whether "time" or "processes" is the subject in "that would normally be manually done"